gcloud-java-core/src/main/java/com/google/gcloud/BasePage.java

```diff
@@ -22,9 +22,9 @@
 import java.util.Objects;
 
 /**
- * Base implementation for Google Cloud storage list result.
+ * Base implementation for Google Cloud paginated results.
  */
-public class BaseListResult<T extends Serializable> implements ListResult<T>, Serializable {
+public class BasePage<T extends Serializable> implements Page<T>, Serializable {
 
   private static final long serialVersionUID = -6937287874908527950L;
```

Comment:
I'm far from a serialization expert; is there any reason this would need to be changed?